client/web/src/repo/RepoRevisionSidebarSymbols.tsx

```diff
@@ -26,28 +26,17 @@ import { parseBrowserRepoURL } from '../util/url'
 
 import styles from './RepoRevisionSidebarSymbols.module.scss'
 
-function symbolIsActive(symbolLocation: string, currentLocation: H.Location): boolean {
-    const current = parseBrowserRepoURL(H.createPath(currentLocation))
-    const symbol = parseBrowserRepoURL(symbolLocation)
-    return (
-        current.repoName === symbol.repoName &&
-        current.revision === symbol.revision &&
-        current.filePath === symbol.filePath &&
-        isEqual(current.position, symbol.position)
-    )
-}
-
 const symbolIsActiveTrue = (): boolean => true
 const symbolIsActiveFalse = (): boolean => false
 
 interface SymbolNodeProps {
     node: SymbolNodeFields
-    location: H.Location
     onHandleClick: () => void
+    isActive: boolean
 }
 
-const SymbolNode: React.FunctionComponent<SymbolNodeProps> = ({ node, location, onHandleClick }) => {
-    const isActiveFunc = symbolIsActive(node.url, location) ? symbolIsActiveTrue : symbolIsActiveFalse
+const SymbolNode: React.FunctionComponent<SymbolNodeProps> = ({ node, onHandleClick, isActive }) => {
+    const isActiveFunc = isActive ? symbolIsActiveTrue : symbolIsActiveFalse
```

Comment:
Unsure why `NavLink` doesn't just accept a `boolean`

```suggestion
    const isActiveFunc = () => isActive
```